src/test/articlesTest.js

```diff
@@ -1,12 +1,24 @@
 import chai from 'chai';
 import chaiHttp from 'chai-http';
 import app from '../index';
+import models from '../models';
+import Tokenizer from '../helpers/tokenGenerator';
+
+const { User, Articles } = models;
+const { generateToken } = Tokenizer;
 
 chai.use(chaiHttp);
 chai.should();
 
+
 let tokens = ' ';
```

Comment:
You can initialize all the variables in one line e.g let tokens = ' ', userObject, articleObject, testUser, testArticle, tokenGen;